core/tests/test_window.py

```diff
@@ -354,6 +354,52 @@ def test_as_image(window):
     assert image.size == (318, 346)
 
 
+def test_screen(window, app):
+    """A window can be moved to a different screen"""
+    # Cannot actually change window.screen, so just check
+    # the window positions as a substitute for moving the
+    # window between the screens.
+    # `window.screen` will return `Secondary Screen`
+    assert window.screen == app.screens[1]
+    assert window.position == (100, 100)
+    window.screen = app.screens[0]
+    assert window.position == (1466, 868)
+
+
+def test_screen_position(window, app):
+    """The window can be relocated using absolute and relative screen positions."""
+    # _________________________________________________
+    # Display Setup:                                  |
+    # ________________________________________________|
+    #              |--1366--|                         |
+    # (-1366,-768) _________                          |
+    #          |  |         |                         |
+    #         768 |Secondary|                         |
+    #          |  | Screen  |                         |
+    #          |  |_________|(0,0)                    |
+    #                          _________              |
+    #                      |  |         |             |
+    #                    1080 | Primary |             |
+    #                      |  | Screen  |             |
+    #                      |  |_________|(1920,1080)  |
+    #                         |---1920--|             |
+    # ________________________________________________|
+    #  `window.screen` will return `Secondary Screen` |
+    #   as window is on secondary screen to better    |
+    #   test out the differences between              |
+    #   `window.position` & `window.screen_position`. |
+    # ________________________________________________|
```

Comment:
This detail is in the dummy backend; no need to repeat it here - at best, a reference to the fact that details about the screen layout can be found in the dummy implemenation would be sufficient.

Reply:
Done.